packages/markdown-docx/test/data/ciceroMark/nested.json

```diff
@@ -0,0 +1 @@
+{"$class":"org.accordproject.commonmark.Document","xmlns":"http://commonmark.org/xml/1.0","nodes":[{"$class":"org.accordproject.commonmark.Heading","level":"2","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"Try TemplateMark"}]},{"$class":"org.accordproject.commonmark.Paragraph","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"Just trying "},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"few things "},{"$class":"org.accordproject.commonmark.Emph","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"here"}]}]},{"$class":"org.accordproject.commonmark.Softbreak"},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"hello"}]},{"$class":"org.accordproject.commonmark.Text","text":" "},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"there"}]}]}]}
```

Comment:
Is this test doing something different than the test you proposed in #418 ?

Reply:
@algomaster99 
it can explain why ternary operator is used in `line 195`.

Reply:
Are you talking about [this line](https://github.com/accordproject/markdown-transform/pull/421/files#diff-f99161f5498dc7c658d94fdc1be57273146a954b6b0eb0359998d748d97cc9f1R195) (`} else if (runTimeProperties.name === 'w:b') {`)? This doesn't have a ternary operator.

Reply:
` nodeInformation.value = runTimeNodes.elements ? runTimeNodes.elements[0].text : ' ';`
This line. I think it got shifted a bit in the latest commit.

Reply:
Check [this step](https://github.com/accordproject/markdown-transform/runs/2979775064#step:6:1121) out. This is from the build of 00f483038c63756bf377d045a213364227dcfeeb - the commit just before you added this test case. The line which you are mentioning was already covered by the coverage tool so adding this test case is not useful for checking this line. Is it doing something else?

Reply:
The test is a special case. It does not increase coverage,  but is there to tell why ternary operator is required and not simply write the expression `runTimeNodes.elements[0].text`

Reply:
Okay, now it makes sense.